src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -298,17 +298,42 @@ public void Initialize()
             }
         }
 
-        public async Task PutEditVersioningCommandAsync(JsonOperationContext context, string databaseName, BlittableJsonReaderObject val)
+        public async Task<long> ModifyDatabaseExpirationBundle(TransactionOperationContext context, string name, BlittableJsonReaderObject configurationJson)
```

Comment:
there is no use to create command just so we serialize it back into json reader. while it doesn't look dense it is much more performent. I have only modified the endpoint that were sharing duplicate code, Generating the same dynamic value structure doesn't justify adding more abstraction layersץ